examples/package.json

```diff
@@ -1,39 +1,28 @@
 {
-    "name": "examples",
-    "version": "0.1.0",
-    "private": true,
+    "name": "example",
+    "version": "1.0.0",
+    "main": "index.js",
+    "license": "MIT",
+    "scripts": {
+        "start": "parcel index.html",
+        "build": "parcel build index.html"
+    },
     "dependencies": {
-        "@types/jest": "^24.0.21",
-        "@types/node": "^12.12.5",
-        "@types/react": "^16.9.11",
-        "@types/react-dom": "^16.9.3",
-        "@types/react-router-dom": "^5.1.2",
-        "react": "^16.11.0",
-        "react-dom": "^16.11.0",
+        "react-app-polyfill": "^1.0.0",
         "react-router-dom": "^5.1.2",
-        "react-scripts": "3.2.0",
-        "react-use-audio-player": "^0.0.10",
-        "typescript": "^3.7.2"
+        "react-use-audio-player": "^0.0.10"
     },
-    "scripts": {
-        "start": "react-scripts start",
```

Comment:
we can remove the README in the examples folder that was generated by create-react-app